README.md

```diff
@@ -62,7 +67,7 @@ Install this project with: cd awesomebooks-project npm install
 
 ## 🤝 Contributing <a name="contributing"></a>
 
-Contributions, issues, and feature requests are welcome! And here is the link to the "issues" section:https://github.com/mariegrace31/awesomebooks-project/issues
+Contributions, issues, and feature requests are welcome! And here is the link to the "issues" section: https://github.com/mariegrace31/awesomebooks-project/issues
```

Comment:
- [OPTIONAL] The issues link is working 🥇 How about using something like below for providing issues page link to increase readability 💪✨

  > _Contributions, issues, and feature requests are welcome! Feel free to check the [issues page](https://github.com/mariegrace31/awesomebooks-project/issues)_